core-ui/src/components/Predefined/Details/StorageClasses/StorageClasses.details.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { LayoutPanel } from 'fundamental-react';
+import { LayoutPanelRow } from 'shared/components/LayoutPanelRow/LayoutPanelRow';
+import { PersistentVolumesList } from 'shared/components/PersistentVolumesList';
+import { PersistentVolumeClaimsList } from 'shared/components/PersistentVolumeClaimsList';
+
+import { EMPTY_TEXT_PLACEHOLDER } from 'react-shared';
+
+export const StorageClassesDetails = ({ DefaultRenderer, ...otherParams }) => {
+  const { t } = useTranslation();
+
+  const StorageClassParameters = storageclass => {
+    const parameters = storageclass?.parameters || [];
+
+    return (
+      <LayoutPanel className="fd-margin--md" key={'storageclass-parameters'}>
+        <LayoutPanel.Header>
+          <LayoutPanel.Head title={t('storage-classes.headers.parameters')} />
+        </LayoutPanel.Header>
+        <LayoutPanel.Body>
+          {Object.keys(parameters).length > 0 ? (
+            Object.entries(parameters).map(parameters => {
+              return (
+                <LayoutPanelRow name={parameters[0]} value={parameters[1]} />
+              );
+            })
+          ) : (
+            <p className="no-entries-message body-fallback">
+              {t('common.messages.no-entries-found')}
+            </p>
+          )}
+        </LayoutPanel.Body>
+      </LayoutPanel>
+    );
+  };
+
+  const customColumns = [
+    {
+      header: t('storage-classes.headers.provisioner'),
+      value: ({ provisioner }) => (
+        <p>{provisioner ? provisioner : EMPTY_TEXT_PLACEHOLDER}</p>
```

Comment:
maybe just
```suggestion
        <p>{provisioner || EMPTY_TEXT_PLACEHOLDER}</p>
```
same for the reclaimPolicy